File: schemaorgphp/classes/data_type.php
<?php
namespace SchemaOrg;

// DataType see : https://schema.org/DataType
class DataType implements \JsonSerializable {

	public static $context = 'http://schema.org';
	public static $type    = 'DataType';

	/**
	 * With properties from Rdfs:Class see : https://schema.org
	 */


	public function jsonSerialize() {
		$out = array(
			'@context' => 'http://schema.org',
			'@type'    => 'DataType',
		);

		return $out;
	}
}

File: schemaorggo/data_type.go
package schemaorggo

import "encoding/json"

// DataType see : https://schema.org/DataType
type DataType struct {

	// With properties from Rdfs:Class see : https://schema.org
	//

}

func (v DataType) MarshalJSON() ([]byte, error) {
	type Alias DataType

	b, err := json.Marshal((Alias)(v))
	if err != nil {
		return nil, err
	}

	return append([]byte("{\"@context\":\"http://schema.org\",\"@type\":\"DataType\","), b[1:]...), nil
}
